src/pilot/register_overcloud.py

```diff
@@ -257,18 +262,15 @@ def _register_node(self, role, node_ip):
                 sys.exit(1)
 
             # The node isn't registered, so register it
-            self.logger.info("Registering {} {} with CDN".format(role,
-                                                                 node_ip))
+            self.logger.info("Registering {} {}".format(role,
+                                                        node_ip))
 
-            # If we're using an activation key, then construct the args for
+            # If we're using satellite, then construct the args for
```

Comment:
Could consider pulling up "that" from the following line.

Reply:
done